lib/googlesheets/archivetogooglesheets_test.go

```diff
@@ -0,0 +1,99 @@
+package googlesheets
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func Test_findRowPositionAndWhetherToInsertARow(t *testing.T) {
+
+	type testCase struct {
+		name          string
+		values        [][]interface{}
+		checkName     string
+		wantRow       int
+		wantInsertRow bool
+	}
+
+	testCases := []testCase{
+		{
+			name:          "all empty",
+			values:        [][]interface{}{},
+			checkName:     "Anything",
+			wantRow:       0,
+			wantInsertRow: false,
+		},
+		{
+			name:          "empty row",
+			values:        [][]interface{}{[]interface{}{}},
```

Comment:
GoLand indicates that the type is redundant. There are several places like that in this file.
```suggestion
			values:        [][]interface{}{{}},
```